src/scout_apm/core/config/config.py

```diff
@@ -153,13 +158,17 @@ def __init__(self):
         self.core_agent_dir = '/tmp/scout_apm_core'
         self.core_agent_version = 'latest'
         self.defaults = {
+                'app_server': '',
                 'core_agent_dir': self.core_agent_dir,
                 'core_agent_download': True,
                 'core_agent_launch': True,
                 'core_agent_version': self.core_agent_version,
                 'download_url': 'https://s3-us-west-1.amazonaws.com/scout-public-downloads/apm_core_agent/release',
-                'git_sha': '',
+                'framework': '',
+                'framework_version': '',
+                'git_sha': GitRevision().detect(),
```

Comment:
@cschneid - I believe we have a one-to-one match between config settings keys and env vars in all of our agents. This is nice. In this case, it looks like the config setting is `git_sha` and the env var is `SCOUT_REVISION_SHA`. 

I'd suggest we change the config to `revision_sha`. We use this naming for Ruby.

/cc @dlanderson

Reply:
correct. We should use `revision_sha` for the setting. The `git_sha` is a field in `app_server_load` payloads

Reply:
FYI - this change was made after I merged this PR - d92227354fc3eeabf463c3bd01e667742d29c30b